autogpts/autogpt/.env.template

```diff
@@ -235,3 +235,12 @@ OPENAI_API_KEY=your-openai-api-key
 # AP_SERVER_PORT=8000
 # # AP_SERVER_DB_URL - Specifies what connection url the agent protocol database will connect to (Default: Internal SQLite)
 # AP_SERVER_DB_URL=sqlite:///data/ap_server.db
+
+
+################################################################################
+### White/Black Listing URLS
+################################################################################
+##WEB_POLICY - Specifies whether the url list is black list(0) or a whilte list(1)
+#WEB_POLICY=0
```

Comment:
I think we need a policy for users who want to continue using AutoGPT as normal. This looks like it forces them to choose between whitelisting and blacklisting, which some users might not want either. There needs to be a default option that simply disables this feature.